docs/reference/MOLD_LINKER_QUICK_REFERENCE.md

```
@@ -0,0 +1,350 @@
+
+---
+
+## Configuration
+
+### Current State
+
+File: `/Users/kooshapari/CodeProjects/Phenotype/repos/.cargo/config.toml`
+
+```toml
+[build]
+incremental = true
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)